IBAnimatable/TransitionFromDirection.swift

```diff
@@ -25,6 +27,8 @@ public enum TransitionFromDirection {
     case .Bottom:
       // The actual transition direction is oposite, need to reverse
       return kCATransitionFromTop
+    default:
+      return ""
```

Comment:
Not sure that makes sense, I mean `Forward` and `Backward` direction will never use any strings since its just for CA. So, returning `Left` by default can be confusing, isn't it?